blocks/init/src/blocks/components/menu/variations/menu-horizontal.scss

```diff
@@ -0,0 +1,44 @@
+$menu-horizontal: (
```

Comment:
blocks/init/src/blocks/components/menu/variations/menu-horizontal.scss

this component is inside a variations folder and in blocks/init/src/blocks/components/menu/menu-style.scss i don't see any variations folder. Please rename the folders like in the comment 🆙